### ISSUE TITLE
[Accessibility]: appdash Go module dependancy

### ISSUE THREAD

**Issue by mcwalrus** [sourcegraph/sourcegraph#59050]
### Problem description

Paths to sourcegraph.com/sourcegraph/appdash and sourcegraph.com/sourcegraph/appdash-data are no longer available.

This is causing build error with external tool https://github.com/99designs/gqlgen - see [issue #2862](https://github.com/99designs/gqlgen/issues/2862).

### Expected behavior

On command:
```
go get github.com/99designs/gqlgen-contrib
```

to run without error. Instead:
```
go: github.com/99designs/gqlgen-contrib@upgrade (v0.1.0) indirectly requires sourcegraph.com/sourcegraph/appdash-data@v0.0.0-20151005221446-73f23eafcf67: unrecognized import path "sourcegraph.com/sourcegraph/appdash-data": reading https://sourcegraph.com/sourcegraph/appdash-data?go-get=1: 404 Not Found
```

This is returning missing URL 404 on paths.

### Additional details

This is blocking our build process for development and production. 

It would be great if this could be priority to resolve.

Thanks

**Comment by Karl-Sparticle** [comment]
any way to resolve it?

**Comment by mallsk23** [comment]
Are there plans to fix this issue, experiencing the same when trying to pull in:

`sourcegraph.com/sourcegraph/appdash-data@v0.0.0-20151005221446-73f23eafcf67`
